src/cmd/troubleshoot/endpoint.go

```diff
@@ -0,0 +1,40 @@
+package troubleshoot
+
+func getOneAgentImageEndpoint(troubleshootCtx *troubleshootContext) string {
+	imageEndpoint := troubleshootCtx.dynakube.ApiUrlHost() + "/linux/oneagent"
+
+	customImage := troubleshootCtx.dynakube.CustomOneAgentImage()
+	version := troubleshootCtx.dynakube.Version()
+
+	if customImage != "" {
+		imageEndpoint = customImage
+	} else if version != "" {
+		imageEndpoint = imageEndpoint + ":" + version
+	}
```

Comment:
```suggestion
	if customImage != "" {
		return customImage
	}
	return imageEndpoint + ":" + version
```
wouldn't this be more explicit? log could still be done outside of this function

Reply:
if the version is empty this would concatenate a ":" and an empty string